scripts/test-ai-system.py

```
@@ -0,0 +1,123 @@
+            "user_tier": "premium",
+        },
+    ]
+
+    for i, test_case in enumerate(test_cases, 1):
+        print(f"\n🧪 Test {i}: {test_case['name']}")
+        print("-" * 40)
+
+        try:
+            # Analyze complexity
+            message = str(test_case["message"])
+            context: dict[str, Any] = dict(test_case["context"])  # Convert to dict for mypy
```

Flake8 (7.3.0)

[error] undefined name 'Any'

(F821)